### DISCUSSION TITLE
entity-framework-4-0-a-fresh-start-with-demo-application/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#35] [Blog comments]
# entity-framework-4-0-a-fresh-start-with-demo-application/

https://blogs.taiga.nl/martijn/posts/entity-framework-4-0-a-fresh-start-with-demo-application/

**Comment by disqus-comments-migrator** [comment]
<em>Bas ter Vrugt commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at december 16 2009, 08:07</em>

---

<p>good example.<br>generic repository registration can be done with the Generic Component Registration API.<br>more info:<br><a href="http://blog.lozanotek.com/archive/2009/12/15/Injecting_Your_Favorite_IoC_Into_MVC_Turbine.aspx" rel="nofollow noopener" title="http://blog.lozanotek.com/archive/2009/12/15/Injecting_Your_Favorite_IoC_Into_MVC_Turbine.aspx">http://blog.lozanotek.com/archive/2009/12/15/Injecting_Your_Favorite_IoC_Into_MVC_Turbine.aspx</a></p>
